lib/src/model.dart

```diff
@@ -1789,8 +2014,9 @@ abstract class ModelElement implements Comparable, Nameable, Documentable {
       params = (element as FunctionTypeAliasElement).parameters;
     }
 
-    _parameters =
-        params.map((p) => new Parameter(p, library)).toList(growable: false);
+    _parameters = params
+        .map((p) => new ModelElement.from(p, library))
+        .toList(growable: false);
```

Comment:
Is `growable: false` being used for immutability? There are some immutable collections in dart:collections I believe.